pkgs/by-name/pr/pros-cli/package.nix

```
@@ -0,0 +1,52 @@
+{
+  fetchFromGitHub,
+  gcc-arm-embedded,
+  python3Packages,
+  lib,
+}:
+let
+  version = "3.5.5";
+  pname = "pros-cli";
+in
+python3Packages.buildPythonApplication {
+  inherit pname version;
```

having the pname in the `let in` is unnecessary
 ```suggestion
in
python3Packages.buildPythonApplication {
  inherit version;
  pname = "pros-cli";
```